website/docs/r/ecs_service.html.markdown

```diff
@@ -70,7 +70,7 @@ The following arguments are supported:
 * `desired_count` - (Required) The number of instances of the task definition to place and keep running
 * `launch_type` - (Optional) The launch type on which to run your service. The valid values are `EC2` and `FARGATE`. Defaults to `EC2`.
 * `cluster` - (Optional) ARN of an ECS cluster
-* `iam_role` - (Optional) The ARN of IAM role that allows your Amazon ECS container agent to make calls to your load balancer on your behalf. This parameter is only required if you are using a load balancer with your service.
+* `iam_role` - (Optional) ARN of the IAM role that allows Amazon ECS to make calls to your load balancer on your behalf. This parameter is required if you are using a load balancer with your service, but only if your task definition does not use the awsvpc network mode. If using awsvpc network mode, do not specify this role. If your account has already created the Amazon ECS service-linked role, that role is used by default for your service unless you specify a role here.
```

Comment:
do we want to quote `awsvpc` here?

Reply:
Sounds good, its monospace typed in the ECS documentation here: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-networking.html

I'll also fix `network_configuration`'s usage of `awsvpc` as well.